File: 2020/skel/skel.py
import math
import sys


class Task:
    def read(self, file='in'):
        with open(file, 'r') as f:
            v = [int(line) for line in f.readlines()]
        return v

    def compute(self, v):
        pass

    def print(self, result):
        print('{}\n{}\n'.format(result[0], result[1]))

    def solve(self):
        v = self.read()
        self.print(self.compute(v))


Task().solve()

File: 2020/skel/skel.cpp
#include "task_runner.h"

struct Task {
    int n;
    std::vector<int> v;

    auto solve() {
        read();
        print(compute());
    }

    void read() {
        // TODO: read from STDIN
        int x;
        while (std::cin >> x) {
            ++n;
            v.push_back(x);
        }
    }

    std::pair<uint64_t, uint64_t> compute() {
        // TODO: computes results
        return {
            1ULL, // TODO: change me
            1ULL, // TODO: change me
        };
    }

    void print(const auto& result) {
        // TODO: print results to STDOUT
        std::cout << result.first << "\n";
        std::cout << result.second << "\n";
    }
};

int main() {
    TaskRunner<Task>::create()->set_stream("in", std::cin).solve();
    // TaskRunner<Task>::create()->set_stream("in", std::cin).set_stream("out", std::cout).solve();
    return 0;
}
